mono/profiler/proflog.c

```diff
@@ -2490,6 +2541,517 @@ counters_and_perfcounters_sample (MonoProfiler *prof)
 	perfcounters_sample (prof, (now - start) / 1000/ 1000);
 }
 
+#define COVERAGE_DEBUG(x) if (debug_coverage) {x}
+static GHashTable *coverage_methods = NULL;
```

Comment:
Oh, ok, I thought I remembered someone say it wasn't possible to do but I must have been mistaken. I'll add this feature